zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/batchoperation/BatchOperationStateTest.java

```diff
@@ -59,12 +60,17 @@ void shouldCreateBatchOperation() throws JsonProcessingException {
             .build();
     final String username = "bud spencer";
     final var authentication = CamundaAuthentication.of(b -> b.user(username));
+    final var authorizationCheck =
+        Authorization.withAuthorization(
+            Authorization.of(a -> a.processDefinition().updateProcessInstance()), "myProcess");
     final var record =
         new BatchOperationCreationRecord()
             .setBatchOperationKey(batchOperationKey)
             .setBatchOperationType(type)
             .setEntityFilter(new UnsafeBuffer(MsgPackConverter.convertToMsgPack(filter)))
-            .setAuthentication(new UnsafeBuffer(MsgPackConverter.convertToMsgPack(authentication)));
+            .setAuthentication(new UnsafeBuffer(MsgPackConverter.convertToMsgPack(authentication)))
+            .setAuthorizationCheck(
+                new UnsafeBuffer(MsgPackConverter.convertToMsgPack(authorizationCheck)));
```

Comment:
💭 Setting it here ensures we can still persist such records in the state without issue. The persisted batch operations don't need and thus don't contain the (potentially) defined permissions to check.